### PR TITLE
Panning and Zooming

### PR DESCRIPTION
**Pull request by notthatmj**

### CODE REVIEW

File: Hex Paper/HexGridViewController.swift [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+import UIKit
+
+class HexGridViewController: UIViewController {
+
+    @IBOutlet weak var hexGridView: HexGridView!
+    @IBOutlet weak var scrollView: UIScrollView!
+    @IBOutlet weak var hexGridViewBottomConstraint: NSLayoutConstraint!
+    @IBOutlet weak var hexGridViewLeadingConstraint: NSLayoutConstraint!
+    @IBOutlet weak var hexGridViewTopConstraint: NSLayoutConstraint!
+    @IBOutlet weak var hexGridViewTrailingConstraint: NSLayoutConstraint!
```
Comment (jacobvanorder) [inline Hex Paper/HexGridViewController.swift]: `@IBOutlets` should be `strong` and by that, I mean `weak`

**Comment by jacobvanorder** [comment]
No huge issues. I haven't used a scroll view for zooming a view in like 6 years. BLAST FROM THE PAST.

**Review by jacobvanorder — APPROVED** [review]

**Comment by notthatmj** [comment]
Fun fact: it took me *forever* to get the scroll view working with auto layout. Turns out I had a constraint reversed (among other problems). Anyhow, I changed the `@IBOutlets` to strong references.

**Comment by jacobvanorder** [comment]
It's so shitty that they have a tech document:
https://developer.apple.com/library/content/technotes/tn2154/_index.html

**Pull request merged**